README.md

```diff
@@ -149,6 +149,8 @@ $ mpirun -np 16 \
     python train.py
 ```
 
+3. To run on Kubernetes, see [Kubeflow](https://github.com/kubeflow/kubeflow/tree/master/kubeflow/openmpi)
+
 ## Keras
```

Comment:
Since you're refactoring it this way, can you make running with Docker #3 (it's currently just in the paragraph above), and Kubeflow #4?

Reply:
Done. I also added TOC. See https://github.com/jiezhang/horovod/blob/patch-1/README.md